src/test/java/uk/gov/hmcts/cmc/claimstore/services/CallbackHandlerFactoryTest.java

```diff
@@ -80,7 +80,7 @@ public void shouldDispatchCallbackForMoreTimeRequested() {
 
         CallbackRequest callbackRequest = CallbackRequest
             .builder()
-            .eventId(MORE_TIME_REQUESTED_PAPER.getValue())
+            .eventId(RESPONSE_MORE_TIME.getValue())
```

Comment:
Should this have changed? Don't we have 2 events, one for citizen requesting more time and one for caseworker?

Reply:
I feel MORE_TIME_REQUESTED_PAPER and RESPONSE_MORE_TIME are same thing, case worker will receive paper or call too update the response deadline, however would be nice to get clarification on it